### ISSUE TITLE
Include PrintExtendedThreadInfo to the list of default JVM options 

### ISSUE THREAD

**Issue by medvedevigorek** [pravega/pravega-operator#460]
### Description

Since JRE11 additional fields are printed for each in a thread dump. These fields are `cpu`, `elapsedTime`, `allocated` and `defined_classes`.

Only first two are printed by default, the other fields must be explicitly enabled by a JVM option 

More details can be found here -> https://bugs.openjdk.java.net/browse/JDK-8200720 
### Importance

This additional information is useful during the service actions when a thread dump is generated 

### Location

Charts/values files 

### Suggestions for an improvement

Include  `-XX:+PrintExtendedThreadInfo` to the list of default JVM options for both Controller and SegmentStore

**Comment by anishakj** [comment]
I think we have to add this option in charts while we publish pravega  `0.9.0` . This JVM option should not be specific to operator version.

**Comment by medvedevigorek** [comment]
Agree.  Ideally we shouldn't have many JVM options hard coded in the operator code and have them provided via the charts as it makes the whole solution more flexible. At the same time we need to set the default flags in charts

**Comment by Ranganaths8** [comment]
@medvedevigorek JVM options are customizable on operator side.. We do set default values for this parameters.

**Comment by anishakj** [comment]
Fixed by PR https://github.com/pravega/pravega-operator/pull/546
